pom.xml

```diff
@@ -1078,6 +1078,27 @@
                     </reportPlugins>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>animal-sniffer-maven-plugin</artifactId>
+                <version>1.14</version>
```

Comment:
Ok to activate 1.15
Please note it does not solve the NPE observed : 
 - data-jpa
 - spring
 - queue-file